### ISSUE TITLE
Default URL is different than forked repo ng-connection-service

### ISSUE THREAD

**Issue by TanayBhalani** [yildiraymeric/ngx-connection-service#3]
Is there any specific reason that default URL to check for internet connectivity in this repo is different than ng-connection-service

Default URL in ng-connection-service
`heartbeatUrl: '//internethealthtest.org',`

Default URL in ngx-connection-service
`heartbeatUrl: '//server.test-cors.org/server?id=' + Date.now() + '&enable=true&status=200&credentials=false',`

**Comment by yildiraymeric** [comment]
Yes "internethealthtest.org" was down and unreachable when I cloned the project. It is working now we can restore to original url

**Comment by TanayBhalani** [comment]
Okay, it seems currently that server is down again
